lib/namedQuery/namedQuery.server.js

```diff
@@ -30,7 +30,12 @@ export default class extends Base {
      * @returns {any}
      */
     getCount() {
+        return this.getCursor().count();
+    }
+
+    // TODO document
+    getCursor() {
         let body = prepareForProcess(this.body, this.params);
-        return this.collection.find(body.$filters || {}, {}).count();
+        return this.collection.find(body.$filters || {}, {});
```

Comment:
this is a count cursor, we need things like { _id: 1 } in fields, otherwise we get a lot of unecessary data when we subscribe to it.